webpack/module-minifier-plugin-5/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "5.1.15",
   "description": "This plugin splits minification of webpack compilations into smaller units.",
   "main": "lib/index.js",
-  "typings": "dist/module-minifier-plugin.d.ts",
+  "typings": "dist/webpack5-module-minifier-plugin.d.ts",
```

Comment:
Should we also change the path in the repo for the package folder? Bit odd that it's `webpack/module-minifier-plugin-5` when this is `webpack5-module-minifier-plugin`

Reply:
Yeah we should just rename these.